controllers/space/MembershipController.php

```diff
@@ -102,14 +102,19 @@ public function actionGetSpaces(){
 
     public function getSpacesListData(Membership $membership){
         return [
-            'user' => UserDefinitions::getUserShort($membership->user),
-            'space_id' => $membership->space_id,
+            'owner' => UserDefinitions::getUserShort($membership->user),
```

Comment:
this is the same value as before, right?

`'user' => UserDefinitions::getUserShort($membership->user),` compared with `'owner' => UserDefinitions::getUserShort($membership->user),`

it is not the owner of the space, or am i wrong?

Reply:
No, 'owner' or 'user' is not related to the space's owner.

Space owner value was associated with originator_user

Reply:
i cannot understand exactly. can you please return the owner always like for `/space`:

```
"owner": {
  "id": 1,
  "guid": "f4122bdab1-3a22ba-4734-b5c0-d3dba5c75",
  "display_name": "A BX,
  "url": "http://...."
}
```

Reply:
Hi @donni106 
Can you explain to me more about this owner key or what you want to show data in this key?

Reply:
the owner `id` and `guid` would be enough if it is not possible to show the same as for `/space`. we need to check if the current user is the owner or not.

Reply:
`$membership->originator` seems to be always `null` in my tests

Reply:
> the owner `id` and `guid` would be enough if it is not possible to show the same as for `/space`. we need to check if the current user is the owner or not.

Then in that case $membership->originator will be best way to show whether the current user is the owner of that space or not

Reply:
but as i said an hour ago:

> $membership->originator seems to be always null in my tests

so how to get the owner data?
it should be always there, not only if the user is the owner, like response in `/space`

Reply:
> `$membership->originator` seems to be always `null` in my tests

You can try with that user which is the owner of one space

Reply:
I did

Reply:
Okay @donni106 
In the owner key, the owner details of that space will be shown

Reply:
i think it works now as expected